codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -657,21 +552,30 @@ public void write(Http2Stream stream, int numBytes) {
          * Called when the writability of the underlying channel changes.
          * @throws Http2Exception If a write occurs and an exception happens in the write operation.
          */
-        public void channelWritabilityChange() throws Http2Exception { }
+        void channelWritabilityChange() throws Http2Exception { }
 
         /**
-         * Called when the state is cancelled outside of a write operation.
+         * Called when the state is cancelled.
          * @param state the state that was cancelled.
          */
-        public void stateCancelled(AbstractState state) { }
+        void stateCancelled(FlowState state) { }
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1186)